lib/models/BucketInfo.js

```diff
@@ -521,6 +533,13 @@ class BucketInfo {
         return this._versioningConfiguration &&
             this._versioningConfiguration.Status === 'Enabled';
     }
+    /**
+     * Get unique id of bucket.
+     * @return {string} - unique id
+     */
+    getUid() {
```

Comment:
Seems like the pattern is to also ship a setter (i.e., `setUid`) for each bucketInfo property.

Reply:
I think this is one setter where we will carefully implement the setter when we need use it throughout the project.